OrbitCore/Capture.h

```diff
@@ -89,6 +89,7 @@ class Capture {
   static std::shared_ptr<Session> GSessionPresets;
   static std::shared_ptr<CallStack> GSelectedCallstack;
   static void (*GClearCaptureDataFunc)();
+  static std::vector<std::shared_ptr<Function>> GSelectedFunctions;
```

Comment:
It is actually the point of ownership.  In the case of local symbols, we only send information about the selected functions to the service and that information is stored in GSelectedFunctions.  It could potentially be transformed to unique_ptrs.  Given the size of this PR, I'll address that in another one.